src/subscription/PaymentMethodInput.js

```diff
@@ -199,6 +202,6 @@ class PaypalInput {
 	}
 }
 
-function isPaypalAssigned(accountingInfo: AccountingInfo) {
+export function isPaypalAssigned(accountingInfo: AccountingInfo): boolean {
```

Comment:
It's not used outside, why it's exported?

Reply:
oh yeah i changed it to use outside but then i unused it and didn't unchange